.flowconfig

```diff
@@ -11,6 +11,10 @@ node_modules/react-native/Libraries/react-native/React.js
 ; "node_modules/react-native" but in the source repo it is in the root
 node_modules/react-native/Libraries/react-native/React.js
 
+; based on https://github.com/facebook/react-native/blob/e028ac7af2d5b48860f01055f3bbacf91f6b6956/.flowconfig#L18
```

Comment:
Is this commend necessary? We always update the file to match upstream, modulo our own modifications.